services/manage/app/styles/app.scss

```diff
@@ -132,3 +132,7 @@ body.transitioning {
 .my-05 {
   margin: 0.5em 0;
 }
+
```

Comment:
Can you add a comment for why this happening? e.g. Bootswatch pulse theme doesn't account for this button style?